src/taipy/core/_orchestrator/_orchestrator.py

```diff
@@ -72,7 +72,7 @@ def submit(
         with cls.lock:
             for ts in tasks:
                 for task in ts:
-                    res.append(cls._submit_task(task, submit_id, callbacks=callbacks, force=force))
+                    res.append(cls._submit_task(task, submit_id, submittable.id, callbacks=callbacks, force=force))  # type: ignore
```

Comment:
Why type ignore?

Reply:
I believe because the id attribute isn't available in the Submittable class

Reply:
Oh probably. Thx.